src/Api/Controllers/EntriesController.cs

```diff
@@ -84,4 +84,32 @@ public EntriesController(ICurrentUserService currentUserService)
         var result = await Mediator.Send(command);
         return Ok(Result<EntryDto>.Succeed(result));
     }
+
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <param name="entryId"></param>
+    /// <param name="request"></param>
+    /// <returns></returns>
+    [RequiresRole(IdentityData.Roles.Employee)]
+    [HttpPut("{entryId:guid}/permissions")]
+    public async Task<ActionResult<Result<EntryPermissionDto>>> ManagePermission(
+        [FromRoute] Guid entryId,
+        [FromBody] ShareEntryPermissionRequest request)
+    {
+        var currentUser = _currentUserService.GetCurrentUser();
+        var command = new ShareEntry.Command
```

Comment:
I don't think this name on the command is good,. But you might argue.

Reply:
We have the same command for physical documents that named `ShareDocument`. So, `ShareEntry`.
If we are going to rename it then please do the same with `ShareDocument`.
Any good name for this?

Reply:
ok, create a rename issue

Reply:
Created: https://github.com/ProFile-org/ProFile-BE/issues/338